tools/debug-ui/README.md

```diff
@@ -0,0 +1,41 @@
+# Nearcore Debug UI
+
+## How to Use
+Clone nearcore, go to this directory, run `npm install` (only needed for first time), and then
+```
+npm start
+```
+
+This will serve the UI at localhost:3000.
```

Comment:
here you say 3000, and then 3030 in line 11.. is that on purpose ?

Reply:
Ah nice catch, thanks.